release/models/platform/openconfig-platform-types.yang

```diff
@@ -303,6 +310,22 @@ module openconfig-platform-types {
       "Component is administratively disabled.";
   }
 
+  identity MASTER {
```

Comment:
Is adding a separate identity for `MASTER` really needed? Couldn't this be simpler with `ACTIVE/BACKUP` instead? I don't see much of a difference between `MASTER` or `ACTIVE` except that it _could_ implicitly indicate that redundancy is configured. However, it would probably be better to have an explicit leaf that reports the redundancy mode.

Reply:
Business logic wise, a supervisor has to be `ACTIVE` first to be considered as `MASTER` or `BACKUP`. If we make a new leaf (say `state/redundant-role`), then implementation has to consider the dependency (e.g. avoid populate the leaf as `BACKUP` when the oper-status actually in `DISABLED` state). That's why I feel like maybe adding `MASTER/BACKUP` in oper-status is better, in terms of simplify things? 

Regarding if we need `MASTER`, I have no strong opinion. I was trying to avoid overloading `ACIVE`. Having `MASTER` implicitly indicate there is a redundancy mode is not bad, I guess? 

+ @aashaikh, @robshakir for comments too.

Reply:
My gut instinct is that having a separate leaf/identities only to indicate active/standby status would be simpler.  Otherwise, if you are planning on keeping on the existing structure then it might be better for MASTER/BACKUP to use ACTIVE as the base identity?

I also wonder whether you might want to use a different name than MASTER, given that term is regarded as being problematic, and there are quite a few alternatives that would also work well: e.g., ACTIVE/STANDBY PRIMARY/SECONDARY?

Also, do you want the model to consider more than one ACTIVE, or more than one STANDBY?  Although, I think that multi-rack systems are not so popular now, and otherwise I suspect that most systems just have an active/standby pair.

Reply:
Thanks for all the comments, and apologize for my late response. 

I have done the following updates based on the feedbacks: 
* Uses a separate leaf to record the redundant state. 
* Uses PRIMARY/SECONDARY wording. 

Marking this as resolve. Can you help review the new changes? Thanks!